Override ckConfigurator for local needs: allow plugin qtiUnderline

jsconan opened the pull request:
depends on : https://github.com/oat-sa/tao-core/pull/329

krampstudio commented on views/js/controller/creator/helpers/ckConfigurator.js:
```diff
@@ -0,0 +1,51 @@
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2014 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
```
Please change the header

krampstudio commented on views/js/controller/creator/helpers/ckConfigurator.js:
```diff
@@ -0,0 +1,51 @@
+     *
+     * @param editor instance of ckeditor
+     * @param toolbarType block | inline | flow | qtiBlock | qtiInline | qtiFlow | reset to get back to normal
+     * @param options is based on the CKEDITOR config object with some additional sugar
```
Better way to document options: 

```javascript
/**
 *  @param {Object} [options] -   is based on the CKEDITOR config object with some additional sugar
 *  @param {Boolean} [options.qtiImage] - enables the qtiImage plugin
 * ...
 */
```

krampstudio commented:
Check also for missing `use strict` and wrong header everywhere

jsconan commented:
I added missing 'use strict'; and fixed wrong headers

2015-05-21 11:45 GMT+02:00 Bertrand CHEVRIER <notifications@github.com>:

> Check also for missing use strict and wrong header everywhere
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oat-sa/extension-tao-testqti/pull/59#issuecomment-104205180>
> .
>



-- 




*Jean-Sébastien CONAN*
Software Engineer

•   Email:   jean-sebastien.conan@vesperiagroup.com
•   GSM: (+352) 621 320 343

*Vesperia Group*
148, Route d'Arlon
L-8010 Strassen
Luxembourg

http://www.vesperiagroup.com/
